src/observability/cilium.go

```diff
@@ -0,0 +1,229 @@
+package observability
+
+import (
+	"reflect"
+
+	"github.com/accuknox/auto-policy-discovery/src/common"
+	"github.com/accuknox/auto-policy-discovery/src/libs"
+	"github.com/accuknox/auto-policy-discovery/src/types"
+	"github.com/cilium/cilium/api/v1/flow"
+	"github.com/golang/protobuf/ptypes/wrappers"
+)
+
+func convertFlowLogToCiliumLog(flowLog *flow.Flow) types.CiliumLog {
+	ciliumLog := types.CiliumLog{}
+
+	if flowLog != nil {
```

Comment:
rather than putting a `!=` condition ... can you use `==` and reduce the nesting.
```suggestion
	if flowLog == nil {
	     return
	}
```